unicodetools/src/main/java/org/unicode/xml/UCDPropertyDetail.java

```diff
@@ -530,16 +588,19 @@ public class UCDPropertyDetail {
                     true,
                     false,
                     false,
-                    true);
-    public static UCDPropertyDetail Grapheme_Link_Detail =
-            new UCDPropertyDetail(
-                    UcdProperty.Grapheme_Link,
-                    VersionInfo.getInstance(3, 2, 0),
-                    59,
                     true,
-                    false,
-                    false,
-                    true);
+                    false);
+// Deprecated
+//    public static UCDPropertyDetail Grapheme_Link_Detail =
+//            new UCDPropertyDetail(
+//                    UcdProperty.Grapheme_Link,
+//                    VersionInfo.getInstance(3, 2, 0),
+//                    59,
+//                    true,
+//                    false,
+//                    false,
+//                    true,
+//                    false);
```

Comment:
i suggest you just delete this altogether

Reply:
I screwed up here - I should have added values for maxVersion, which I have done now.